src/content/app/tools/blast/components/blast-app-bar/BlastAppBar.tsx

```diff
@@ -50,13 +62,27 @@ const ToolsAppBar = () => {
     }
   };
 
-  const speciesTabs = speciesList.map((species, index) => (
+  const enabledSpecies = speciesList.map((species, index) => (
     <SelectedSpecies
       key={index}
       species={species}
       onClick={() => speciesLozengeClick(species)}
     />
   ));
+
+  // TODO: this is not a proper way of expressing disabled species lozenges in this context
```

Comment:
Lets remove the todo and create a jira ticket for this...it wont get done if it doesnt go in a jira.

Reply:
Removed the TODO